libraries/cli/include/cli/default_config.hpp

```diff
@@ -183,7 +183,8 @@ constexpr std::string_view default_json = R"foo({
         },
         "execution_options": {
           "disable_gas_fee": false,
-          "disable_nonce_check": false
+          "disable_nonce_check": false,
```

Comment:
Do we still need disable_nonce_check? I think if enable_nonce_skipping is true, we must have nonce check

Reply:
I think we should keep nonce check value, just in case we want to completely disable this option.